.github/workflows/build_and_test.yml

```diff
@@ -623,6 +623,12 @@ jobs:
     - name: Run documentation build
       run: |
         cd docs
+        if [ -f "./dev/is-changed.py" ]; then
```

Comment:
Looks like the path should be changed relative to `docs`? or move before `cd docs`.

Reply:
Oh.. Right. Thank you for catching.